commcare_connect/program/migrations/0001_initial.py

```diff
@@ -31,9 +32,41 @@ class Migration(migrations.Migration):
                     "delivery_type",
                     models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to="opportunity.deliverytype"),
                 ),
+                (
+                    "organization",
```

Comment:
maybe this is addressed in a later PR, but I thought we talked about no longer making this nullable and having all the program views be org specific

Reply:
Added https://github.com/dimagi/commcare-connect/pull/379/commits/4d300db6ef63716bfe752b9952f5f92c471d5bb7